data/projects/all.json

```diff
@@ -235,9 +235,52 @@
             "sonatype_lifecycle": false
         },
         "tags": [
-            "nexus",
-            "repository",
             "blobstore"
         ]
+    },
+    {
+        "name": "API Clients",
```

Comment:
I don't want to commit marketing blasphemy, but I find these two images "too similar" to be clear. Could differentiate the products in the title, something like: "Nexus Repo API Clients", and "IQ Server API Clients"? Any short product name in front of "API Clients" would suffice: (NXRM, IQ even?)

<img width="1089" alt="Screenshot 2024-11-14 at 12 02 03 PM" src="https://github.com/user-attachments/assets/ed041c5d-7177-43fd-b840-6a7b5ea9ae8b">

Reply:
@bhamail - they were previously but the longer names borked the layout formatting...

Open to suggestions